src/test/java/duke/misc/ParserTest.java

```diff
@@ -0,0 +1,23 @@
+package duke.misc;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import duke.exception.InvalidDescriptionException;
+import duke.exception.InvalidTypeException;
+
+public class ParserTest {
+    @Test
+    public void invalidCommandTest() {
+        try {
+            Parser.handleInput("example hello");
+        } catch (InvalidTypeException e) {
+            assertEquals("OOPS!!! I'm sorry, but I don't know what that means :-(",
+                    e.getMessage());
+        } catch (InvalidDescriptionException e) {
+            assertEquals("OOPS!!! The description of a task cannot be empty",
+                    e.getMessage());
```

Comment:
I really like how you're also testing your exceptions! But perhaps you could also include some test cases for normal expected inputs?